app/README.md

```
@@ -7,28 +7,39 @@ It currently supports three modes:
 2. Desktop Mode - when Requestly desktop application is launched
 3. Remote - when account is connected to a mobile application
 
+### Prerequisites
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)